src/utils/__tests__/error-utils.unit.spec.js

```diff
@@ -0,0 +1,118 @@
+import { failAction } from '../error-utils.js'
+
+describe('error-utils.unit', () => {
+  describe('failAction', () => {
+    let mockH
+    const defaultMockError = {
+      details: [
+        {
+          message: '"name" is required',
+          path: ['name']
+        },
+        {
+          message: '"age" must be a number',
+          path: ['age']
+        }
+      ],
+      _original: {
+        name: '',
+        age: 'not-a-number'
+      }
+    }
+
+    beforeEach(() => {
+      mockH = {
+        response: jest.fn().mockReturnThis(),
+        code: jest.fn().mockReturnThis(),
+        takeover: jest.fn().mockReturnValue('takeover-result')
+      }
```

Comment:
I appreciate this is resetting the mock, but it's not a pattern that we've used and I think having a `getMockH` function would be more consistent:
```
const getMockH = () => ({
        response: jest.fn().mockReturnThis(),
         code: jest.fn().mockReturnThis(),
         takeover: jest.fn().mockReturnValue('takeover-result')
})
```